src/components/stepindicator/StepIndicator/StepIndicator.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react'
+import classnames from 'classnames'
+import { StepIndicatorStepProps } from '../StepIndicatorStep/StepIndicatorStep'
+
+interface StepIndicatorProps {
+  showLabels?: boolean
+  counters?: true | false | 'small'
+  centered?: boolean
+  children: React.ReactElement<StepIndicatorStepProps>[]
+  className?: string
+  divProps?: JSX.IntrinsicElements['div']
+  listProps?: JSX.IntrinsicElements['ol']
+}
+export const StepIndicator = (
+  props: StepIndicatorProps
+): React.ReactElement => {
+  const {
+    showLabels = true,
+    counters = false,
+    centered = false,
+    children,
+    className,
+    divProps,
+    listProps,
+  } = props
+
+  const classes = classnames(
+    'usa-step-indicator',
+    {
+      'usa-step-indicator--no-labels': !showLabels,
+      'usa-step-indicator--counters': counters === true,
+      'usa-step-indicator--counters-sm': counters === 'small',
+      'usa-step-indicator--center': centered,
+    },
+    className
+  )
+
+  const findCurrentStepIndex = (): number => {
+    const i = children.findIndex((step) => step.props.status === 'current')
+    return i === -1 ? 0 : i
+  }
+  const currentStepIndex = findCurrentStepIndex()
+  const currentStepNumber = currentStepIndex + 1
+  const currentStepLabel = children[parseInt(`${currentStepIndex}`)].props.label
+  const totalNumberOfSteps = children.length
+
+  return (
+    <div className={classes} data-testid="step-indicator" {...divProps}>
```

Comment:
Should we add `aria-label: progress` here since its explicitly called out in the step indicator [guidance?](https://designsystem.digital.gov/components/step-indicator/) it could always be overriden by `divPops`

Reply:
Meant to make a separate commit for this, but it was also fixed in ec8cf08